os/drivers/serial/serial.c

```diff
@@ -886,6 +977,15 @@ static int uart_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 		break;
 
 #ifdef CONFIG_SERIAL_TERMIOS
+		case LOG_DYNAMIC: {
```

Comment:
We have a naming rule for all ioctl.
This violates that policy. please see ioctl.h, but please refer to my above comment in top area.